frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -980,7 +980,7 @@ export const ClusterServiceVersionsDetailsPage: React.FC<ClusterServiceVersionsD
                   href: referenceForProvidedAPI(desc),
                   name: ['Details', 'YAML', 'Subscription', 'Events'].includes(desc.displayName)
                     ? `${desc.displayName} Operand`
-                    : desc.displayName,
+                    : desc.displayName || desc.kind,
```

Comment:
This should probably be broken out into a small helper function. Also, this bypasses the logic that appends ' Operand' to the name if it matches an existing tab name. It might be possible for `desc.kind` to match an existing tab name as well.

Reply:
@TheRealJon Interesting! Since this PR merged, I can address the breaking of logic into small helper function in another PR and BZ. However I'm thinking of where else the function would be useful.  Could you please point me to an operand that the `desc.kind`  match an existing tab name if you know of any?

Reply:
@cyril-ui-developer I don't know of any examples off the top of my head. But these are CRDs, which means they are named by the authors, so we have to assume that they could collide with existing tabs.